common/oe_host_string.h

```diff
@@ -1,6 +1,12 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+/*
+ * There are separate OE functions for strerror and strerror_s.
+ * Therefore we suppress the warning about not using more secure CRT functions.
+ */
+#pragma warning(disable : 4996)
```

Comment:
What will this pragma do on gcc/clang? Should this be ifdef'd to only win32?

Reply:
Yes, you're right. I'll change it so that it will only apply to Windows.